src/parameters/mod.ts

```diff
@@ -1,12 +1,12 @@
-import SchemaTypes from "../schema/schema_types.ts";
+// import SchemaTypes from "../schema/schema_types.ts";
```

Comment:
We could likely remove this file (or the majority of it) since `ParameterVariable` isn't used without workflows